Editor/Tabs/WDTab.cs

```diff
@@ -202,6 +210,71 @@ private void UpdateWDList()
                     .WithAlignSelf(Align.Center)
                     .WithFlex(1, 0, 1)
                     .ChildOf(row);
+
+                VisualElement buttonContainer = new VisualElement()
+                    .WithAlignSelf(Align.Center)
+                    .WithFlex(1, 0, 1)
+                    .ChildOf(row);
+                Button button = new Button(() =>
```

Comment:
I would put this in its own function separately from the layout

Reply:
Done